### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Get Shared Access Signature_ScreenReader: The user is unable to use the spinner function using up/down arrow keys when the voiceover tab focus lands on date and time inside 'Start time' and 'Expiry time'.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/AzureStorageExplorer#5764]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_AzureToolsStorageExplorer_May2022;#Azure Tools Storage Explorer;#MAC;#FTP;#DesktopApp;#WCAG4.2.9;#Voiceover;#AINotInScope;#Regressed:06-17-22;

## Environment Details:
Application Name: Azure Tools Storage Explorer 
Application Version: 1.24.0
Mac Version: macOS Monterey 12.3.1
Screen Reader: Voice over 

## Repro Steps:
1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch Azure Tools Storage Explorer.
3. Sign in to Azure.
4. Tab to 'Explorer' pane tree view.
5. Navigate to any of the file inside "Blob Containers"
6. Right click on any of the file and select 'Get shared access signature'.
7. Tab till 'Start Time' control.
8. Turn on voiceover to and use spinner function.

## Actual:
The user is unable to use the spinner function using up/down arrow keys when the voiceover tab focus lands on date and time inside 'Start time' and 'Expiry time'.

### Same issue can be observed for the following scenario:

1. Manage Access Policies_Add (User is unable to use stepper function for the 'Start Time' 'Expiry Time')
2. Settings_Transfers (Stepper function is not working for SAS duration, Max Transfer rate, Network Concurrency, File Concurrency, Buffer Size)

## Expected:
The user should be able to use the spinner function using up/down arrow keys when the voiceover focus lands on date and time inside 'Start time' and 'Expiry time'.

## User Impact:
Screen reader users will face difficulty in interacting with the available functions for a control.

## Attachment:
![9_Start time+Expiry timeeeeeeee](https://user-images.githubusercontent.com/98393966/169975213-5b7bcee9-f98f-41ce-abde-e35d4d08f6b9.gif)

**Comment by MRayermannMSFT** [comment]
@bbarik06 why do you not just:
1. focus the date time control
2. use up and down arrow keys to adjust stepper values
3. use left and right arrow keys or tab to move between steppers

???

**Comment by bbarik06** [comment]
@MRayermannMSFT Under normal keyboard functioning, all the arrow keys work for the stepper but when the voiceover is turned on the function doesn't work even if we use the keyboard arrow keys.

**Comment by MRayermannMSFT** [comment]
@bbarik06 so to confirm, if you have VoiceOver turned on, you focus the date control, and the only keys you use are tab or arrow keys, it does not work for you? Can you please provide a screen recording of this?

**Comment by bbarik06** [comment]
@MRayermannMSFT Yes if I have voiceover is turned on, I am using the tab keys to the control, and it does not work. I would like to know if it's working fine on your side. Please refer the attachment.

![5764](https://user-images.githubusercontent.com/98393966/174246302-9172b342-1c2d-4b71-8d46-dde16eb14b44.gif)

 **Note:** Please do not close the bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.

**Comment by bbarik06** [comment]
@MRayermannMSFT Could you please post the same video here.

**Comment by MRayermannMSFT** [comment]
@bbarik06 have you had a chance to review the video? Where should we go from here?
